changelog/4552.trivial.rst

```diff
@@ -0,0 +1 @@
+Added a function `_is_coordinate_helioprojective(coordinates)` which checks whether `coordinates.frame` is helioprojective or not.
```

Comment:
```suggestion
Several functions in `sunpy.map.maputils` now properly check if the provided coordinate is in the expected `~sunpy.coordinates.frames.Helioprojective` frame.
```

Reply:
Also rename this file to `4552.bugfix.rst`

Reply:
Is this really a bugfix?

Reply:
Yup, I'd classify as a bug not checking whether the frame is `Helioprojective` before trying to access Helioprpjective components (`.Tx` and `.Ty`).

Reply:
Good point.